rustls/src/crypto/aws_lc_rs/mod.rs

```diff
@@ -197,3 +197,32 @@ mod ring_shim {
         })
     }
 }
+
+/// AEAD algorithm that is used by `mod ticketer`.
+pub(super) static TICKETER_AEAD: &ring_like::aead::Algorithm = &ring_like::aead::AES_256_GCM;
+
+/// Are we in FIPS mode?
+pub(super) fn fips() -> bool {
+    aws_lc_rs::try_fips_mode().is_ok()
+}
+
+#[cfg(test)]
+mod tests {
+    #[cfg(feature = "fips")]
+    #[test]
+    fn default_suites_are_fips() {
+        assert_eq!(
+            super::DEFAULT_CIPHER_SUITES
+                .iter()
+                .filter(|scs| !scs.fips())
+                .count(),
+            0
+        );
+    }
+
+    #[cfg(not(feature = "fips"))]
+    #[test]
+    fn default_suites() {
+        assert_eq!(super::DEFAULT_CIPHER_SUITES, super::ALL_CIPHER_SUITES);
+    }
```

Comment:
I think these tests belong in the next commit? AIUI they'll fail in this commit.